app/src/main/java/org/wikipedia/descriptions/DescriptionEditTutorialActivity.kt

```diff
@@ -19,7 +19,7 @@ class DescriptionEditTutorialActivity : SingleFragmentActivity<DescriptionEditTu
 
     companion object {
         const val SHOULD_SHOW_AI_ON_BOARDING = "showAIOnBoarding"
-        fun newIntent(context: Context, showAIOnBoarding: Boolean): Intent {
+        fun newIntent(context: Context, showAIOnBoarding: Boolean = true): Intent {
```

Comment:
Do we still need this `showAIOnBoarding` parameter?

Reply:
I have a feeling that we might still need to re-disable or re-AB-test this feature in the future, so it might be useful to keep this option available, until we're totally sure that we're keeping this feature permanently.